packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -275,3 +282,44 @@ export class DefaultCrudRepository<T extends Entity, ID>
     return models.map(m => this.toEntity(m));
   }
 }
+
+export class DefaultKVRepository<T extends Model> implements KVRepository<T> {
+  kvModelClass: typeof juggler.KeyValueModel;
+
+  constructor(kvModelClass: typeof juggler.KeyValueModel) {
+    this.kvModelClass = kvModelClass;
+  }
+
+  delete(key: string, options?: Options): Promise<void> {
+    throw new Error('Not implemented');
+  }
+
+  deleteAll(options?: Options): Promise<void> {
```

Comment:
AFAICT, juggler 3.x provides both `delete` and `deleteAll` operations, although they seems like not documented in our apidocs.

- [`KVAO.prototype.delete(key[, options], cb)`](https://github.com/strongloop/loopback-datasource-juggler/blob/be7d97c88586e0889374665fed52b4bc1c894fab/lib/kvao/delete.js)
- [`KVAO.prototype.deleteAll([options, ]cb)`](https://github.com/strongloop/loopback-datasource-juggler/blob/be7d97c88586e0889374665fed52b4bc1c894fab/lib/kvao/delete-all.js)

See the tests for usage examples:
- [test/kvao/delete-all.suite.js](https://github.com/strongloop/loopback-datasource-juggler/blob/be7d97c88586e0889374665fed52b4bc1c894fab/test/kvao/delete-all.suite.js)
- [test/kvao/delete.suite.js](https://github.com/strongloop/loopback-datasource-juggler/blob/be7d97c88586e0889374665fed52b4bc1c894fab/test/kvao/delete.suite.js)